service/gradle/open-api.gradle

```diff
@@ -1,5 +1,5 @@
-
 // OpenAPI/Swagger Server Generation
+def openapiOutputDir = "${buildDir}/openapi"
```

Comment:
Is this defined in the main `service/build.gradle` file already?

Reply:
right, had meant to remove the one in service/build.gradle since it was only used in this file. done that now